src/Authentication/appsettings.json

```diff
@@ -42,7 +42,7 @@
     "EnableDBConnection": true,
     "WorkspacePath": "Persistance/Migration",
     "AuthenticationDbAdminConnectionString": "Host=localhost;Port=5432;Username=auth_authentication_admin;Password={0};Database=local_authentication_testdb",
-    "AuthenticationDbUserConnectionString": "Host=localhost;Port=5432;Username=auth_authentication;Password={0};Database=local_authentication_testdb",
+    "AuthenticationDbUserConnectionString": "Host=localhost;Port=5432;Username=auth_authentication;Password=postgres;Database=local_authentication_testdb",
```

Comment:
You can remove the AuthenticationDbAdminPassword, AuthenticationDbPassword and add the password in the connectionstring